src/App.tsx

```diff
@@ -1,10 +1,67 @@
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { UserWarning } from './UserWarning';
+import { TodoList } from './components/TodoList';
+import { Todo } from './types/Todo';
+import { getTodos } from './utils/todos';
+import { NotificationText, SortCondition } from './types/enums';
+import { Footer } from './components/Footer';
+import { Notification } from './components/Notification';
 
-const USER_ID = 0;
+const USER_ID = '11161';
 
 export const App: React.FC = () => {
+  const [todosFromServer, setTodosFromServer] = useState<Todo[] | null>(null);
+  const [visibleTodos, setVisibleTodos] = useState<Todo[] | null>(null);
```

Comment:
no need to use the state for `visibleTodos`. You can make it a simple variable since it depends on `todosFromServer` and `filterBy`